### ISSUE TITLE
why TF2.0+TRT6 (keras) slower than TF1+keras about 4 times ?

### ISSUE THREAD

**Issue by VitaMusic** [tensorflow/tensorflow#35161]
I'd like to understand the performance issue I faced :
Why TF2.0+TRT6 (keras) slower than TF1+keras about 4 times ?

GPU K80

before : TF1 + keras (as separate package)
after: TF2 + TRT6 integrated with integrated keras

on the same example TF2 is 4 times slower, when I'm waiting acceleration of training/predition in new version with TRT integration.

**Comment by ravikyram** [comment]
@VitaMusic 
Request you to provide simple standalone code to reproduce the issue in our environment, then it is easy for localizing the issue faster. Thanks!

**Comment by VitaMusic** [comment]
I can not share my code for a IP reasons, but you can try any like VGG NN with any image set.

**Comment by jvishnuvardhan** [comment]
@VitaMusic Can you please share a standalone code with a public data. Issue resolution is faster with a simple standalone code. Thanks!

**Comment by VitaMusic** [comment]
I can not share my code for a IP reasons, I shall see what I can do as a code example, but perhaps you can try any like VGG NN with any image set and it will be faster way for benchmarking. I see there are other issues on performance of tf2, they indicated to do eager mode off, but if I do it I'll have errors with dropout layers. I did not try yet the last TF2 RC1 version update, perhaps this problem was solved already ?
From other side, I'd like to have a benchmark script on public data (training and predicion cases) where TF1+keras vs TF2 (keras inside) comparison done, if you can give me it please? I'll try to run it on my servers.
Perhaps it is alse linked to gpu problem, as I recompiled TF2 for K80.

**Comment by jvishnuvardhan** [comment]
@VitaMusic Can you please try recent TF versions (`TF2.2` and `tf-nightly`)? Can you please share a standalone code with a public data. Issue resolution is faster with a simple standalone code. Thanks!

Please feel to close the issue if this was already resolved for you. Thanks!

**Comment by VitaMusic** [comment]
Cause covid I have to standby this problem for sometime. Will test the new
version later. Sorry for this delay.

вт, 2 Июн 2020 г., 2:49 Vishnuvardhan Janapati <notifications@github.com>:
